Document unsupported reason 'virtualized image' (#31971)
home-assistant/home-assistant.github.io: source/more-info/unsupported/virtualization_image.markdown

---

> title: "Virtualized-Image"

Reviewer: ```suggestion
title: "Virtualization in use"
```

Author: Doesn't that title imply that virtualization isn't supported? I see that I can uses spaces here, I thought I couldn't for some reason. But I'm not sure I agree with that title. What about just "Virtualized image"?

Reviewer: Hm, yeah good point. Maybe:

```suggestion
title: "Image not suited for virtualization"
```

Author: After chatting we settled on "Incorrect image for virtualization"

description: "More information on why virtualized image marks the installation as unsupported."
---

## The issue

> Home Assistant OS publishes specific images which support being run on a virtualized
> system. These come with guest tools and other changes to make this easier. The 
> image of Home Assistant OS you are running on your virtualized system is not one
> of these.

Reviewer: ```suggestion
Your Home Assistant OS installation appears to run in a virtualized environment using
a disk image which is not meant to run in a virtualized environment.
 
Home Assistant OS publishes specific images which support being run on a virtualized
system. These are optimized to run in virtual environments, e.g. have guest tools installed
and para-virtualization drivers enabled.
```

---
title: "Incorrect image for virtualization"
description: "More information on why virtualization image marks the installation as unsupported."
---

## The issue

> 

Reviewer 2: this section describes why something is not running. What is the symptom? What is not working, how do I know I'm having this particular issue.

Reviewer: We show a link in the UI to this page. So the user gets redirected here when the issue is present on his system.
